tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -7,11 +7,17 @@
 
 
 class StatusBar(Base):
```

Comment:
as we have ID's to identify the unique objects in the status bar I don't see the point of using a page region 

PageRegion is used best for repettitive objects like `lists`